schema/stateTransition/base.json

```diff
@@ -8,11 +8,22 @@
     "type": {
       "type": "number",
       "enum": [1, 2]
+    },
+    "signaturePublicKeyId": [
```

Comment:
It supposed to be `signaturePublicKeyId: { type: [“integer”, “null”] }` and please revert back validation rules except `multiplyOf`